application/backend/src/main/resources/apps/bulma/components/table/dialog/.content.json

```diff
@@ -0,0 +1,35 @@
+{
+  "sling:resourceType": "wcm/dialogs/dialog",
+  "tabs": {
+    "sling:resourceType": "wcm/dialogs/components/tabs",
+    "generalTab": {
+      "sling:resourceType": "wcm/dialogs/components/tab",
+      "label": "General",
+      "isBordered": {
+        "sling:resourceType": "wcm/dialogs/components/checkbox",
+        "name": "isBordered",
+        "label": "Add borders to all the cells"
```

Comment:
Let's consider changing those labels to be consistent with the last one. I mean something like `Cells borders` or something like that. And this more descriptive text can be added to the `description`.

Reply:
Fixed!